app/controllers/feedback_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class FeedbackController < ApplicationController
+  skip_before_action :authenticate_user!
+  skip_authorization_check
+
+  def new
+    @feedback = Feedback.new
+  end
+
+  def create
+    @feedback = Feedback.new feedback_params
+    if @feedback.valid?
+      submit_feedback
+      redirect_to new_feedback_path, notice: 'Your feedback has been submitted'
```

Comment:
This should redirect to users root path and have a sensible translation key entry aswell

```suggestion
      redirect_to authenticated_root_path, notice: I18n.t('feedback.submitted.success')
```

should also test this in the request specs

Reply:
Have made the suggested change, and amended feature and request specs to test for redirection to authenticated_root_path after successful feedback submission.